README.md

```diff
@@ -22,23 +22,14 @@ To add another ruby version's source to this gem's directory:
 
     $ rake add_source VERSION=2.1.3 PATCHLEVEL=242
 
-add_source can use pre-downloaded .tgz (use TGZ_FILE_NAME to pass it)
-also it can extract patch level from version.h of downloaded sources, so
-PATCHLEVEL variable is optional.
+_Adding released versions_. `add_source` can use a pre-downloaded .tgz file. Use TGZ_FILE_NAME to pass it. Note that the PATCHLEVEL variable is optional because `add_source` can extract the patch level from version.h of the downloaded sources.
 
-_Adding pre-release versions_. For pre-releases PATCHLEVEL should not be provided,
-as it is not present in distribution. To find sources of pre-release versions,
-this gem will look in `Debase::RubyCoreSource::REVISION_MAP` hash. Please add
-respective entry into this hash on adding pre-release version sources. 
+_Adding pre-release versions_. For pre-releases, do not provide the PATCHLEVEL, as it is not present in distribution. To find sources of pre-release versions, this gem will look in the `Datadog::RubyCoreSource::REVISION_MAP` hash. Please add the respective entry into this hash when adding pre-release version sources. 
 
 ## Credits
 
-* @valich for 2.5.0-preview1 headers and src-based ruby support
-* @dirknilius for 2.2.3 headers
-* @andremedeiros for 2.1.1 headers
-* @formigarafa for fixing 2.1.0 headers
+* 
 
```

Comment:
Be careful about removing credits -- these people still worked on the gem so I think it makes sense to still give them kudos; if we're keeping most of the README to keep this section as well?

Reply:
Agreed, this can be a touchy subject, I have two alternative proposals:

1. Put the credits back but prefix them with something like "The following credits are in upstream debase-ruby_core_source readme:".

2. Restore the original file but rename it to README.upstream.md, so that the default file rendered will be our readme. Create a new file called README.md with just the information relevant/specific to our fork. In our README.md say something like "for more information about the library itself please see README.upstream.md". This way the upstream changes to readme will have somewhere to go, they will (hopefully at least) be more easily applied, and perhaps git could even apply them automatically with its rename detection logic. And all upstream contributors retain their credits.

Reply:
I initially removed the credits because these headers will be removed, but you're right - I'm thinking to use Oleg's suggestion of adding a prefix that these credits are in the upstream when adding them back.